src/views/RichWorkspace.vue

```diff
@@ -211,8 +211,12 @@ export default {
 	}
 
 	#rich-workspace::v-deep #editor {
+		overflow: visible !important;
+	}
+
+	#rich-workspace::v-deep .content-wrapper {
 		overflow: scroll !important;
-		max-height: 50vh;
+		max-height: 40vh;
```

Comment:
Reducing the rich workspace height more would box it in a bit much, no? Otherwise the change seems good.

Reply:
I probably should use calc(50vh - 50px) so that it use a similar size has before

Reply:
Is that needed for the fix?

Reply:
The new code doesn't apply the max-height on the editor but on the content instead so using calc(50vh - 50px) make it so that the height of the editor is the same as before